diff-viewer/src/styles.css

```
@@ -573,15 +576,54 @@ body[data-status-only="true"] #loading-layer {
+  width: 56px;
+  height: 56px;
+  border-radius: 16px;
+  background: color-mix(in lab, var(--cmux-diff-fg) 5%, transparent);
+  border: 1px solid color-mix(in lab, var(--cmux-diff-fg) 8%, transparent);
+}
+body[data-status-only="true"] #status:not([data-error="true"]):not([data-pending="true"]) #status-icon::before {
+  content: "";
+  display: block;
+  width: 26px;
+  height: 26px;
+  background-color: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)